### PR TITLE
Feature/19 1,19 2/pro ass/choose model ai

### PR DESCRIPTION
**Pull request by KoYeJoon**
🧸branch
(모델 연구하는 부분이라 바로 main으로 해도 될 것 같았습니다!)
- main ← feature

<br />

📌 기능
- train code 작성
  - ssdmobilenetv2 tensorflow API 활용하여 train 할 수 있도록 코드 작성 (방법은 train/ssdmobilenetv2/README.md에 포함)
  - ssdmobilenetv2 훈련 완료!!
  - ssdmobilenetv2 tensorflow API 활용 시 model export code 추가

- test code, evaluate_mAP 수정
  - model test 시 결과로 나온 box를 기준으로 mAP를 바로 측정할 수 있도록 연결
  - ssdmobilenetv1, ssdmobilenetv2 mAP 측정 시, 
  - ex ) $ python test.py --eval_data dataset/test --ObjectDetection yolov4-tiny
  - validation set (egohand)로 평가하였을 때의 정확도 측정 결과
    - valid_result/에도 저장되어 있습니다!
validation_result

<br />

✔️ 확인해야 할 사항
- custom data labeling 필요 (labelImg)
- model 평가 기준 고려중
  - ssd가 AP가 가장 높지만, ground truth 없는데, 바운딩 박스를 찾는 경우가 있기 때문에 평가 기준 고려 필요

<br />

✏️ 추후 변경될 수 있는 사항
- custom test data 추가 후, 실제 결과를 분석해볼 예정
- ssd용 label 변형 코드 작성 필요
- yolo 용 label 변형 코드 작성 필요
- 결과 분석 후 우선적으로 그 모델을 test-helper-ai에 추가할 예정

- 이후, 가장 잘되는 모델을 통해 fine tuning 진행 예정
- 시간이 남으면, 파라미터 수를 계산해볼 예정

### CODE REVIEW

**Review by ny-woo — APPROVED** [review]
와우...드디어 훈련 끝난건가여 궁금하네여.... 수고많으셨습니다 👍👍

**Review by leeshyeon — APPROVED** [review]
수고하셨습니다!👍🏻

**Review by SongZzi — APPROVED** [review]
드디어 훈련 끝난건가요.!! 수고많으셨습니다👍👍

**Review by jmchoi1225 — APPROVED** [review]
수고하셨습니다!
